lib/team_collection.rb

```diff
@@ -0,0 +1,17 @@
+require 'csv'
+require './lib/teams.rb'
+
+class TeamCollection
+  attr_reader :total_teams
```

Comment:
I do not believe we need to make total_teams an attr_reader. If it follows a similar pattern to GameCollection, then we will only use the total_teams variable within this class.

Reply:
Interesting... Let's note this somewhere for future.

Reply:
I don't disagree with you but we used an attr_reader for total_games in the GameCollection. We should be consistent between the classes. We should probably remove these for now and readd them if the need arises later.

Reply:
I looked into this more, we do need the attr_reader for our tests to test_it_has_total_teams and test_it_has_total_games. Seems like we need these.